ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WatchedImages/WatchedImagesTable.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { CSSProperties } from 'react';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import { Bullseye, Button } from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+
+import { WatchedImage } from 'types/image.proto';
+import { UseRestMutationReturn } from 'hooks/useRestMutation';
+import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
+import { Empty } from 'services/types';
+
+export type WatchedImagesTableProps = {
+    className?: string;
+    style?: CSSProperties;
+    watchedImages: WatchedImage[];
+    unwatchImage: UseRestMutationReturn<string, Empty>['mutate'];
+    isUnwatchInProgress: boolean;
+    'aria-labelledby'?: string;
+};
+
+function WatchedImagesTable({
+    className,
+    style,
+    watchedImages,
+    unwatchImage,
+    isUnwatchInProgress,
+    ...props
+}: WatchedImagesTableProps) {
+    return (
+        <div className={className} style={style}>
+            {watchedImages.length === 0 && (
+                <Bullseye>
+                    <EmptyStateTemplate title="No watched images found" headingLevel="h2" />
+                </Bullseye>
+            )}
+            {watchedImages.length > 0 && (
+                <TableComposable
+                    aria-labelledby={props['aria-labelledby']}
```

Comment:
I wasn't crazy about this pattern either, but since we can't destructure the `aria-labelledby` prop due to the hyphen it seemed like the three choices were:
1. `...props`
2. Move the destructuring from the function declaration to immediately inside the component, and then access `aria-labelledby` off the `props` argument. Mostly similar to 1 in my opinion.
3. Change the prop name to `ariaLabelledby`, but that felt like it was going against the React pattern of having all of the `aria` props hyphenated.